.github/workflows/zeebe-medic-benchmarks.yml

```diff
@@ -1,85 +0,0 @@
-name: Zeebe Weekly medic benchmark
```

Comment:
❓ Any particular reason why this is deleted? It didn't run anyway as it's not on main, just wondering why you chose to remove it.